src/yadmark/benchmark/runner.py

```diff
@@ -92,26 +92,28 @@ def run_external(self, theory, ocard, pdf):
         observable = ocard
 
         if self.external.upper() == "APFEL":
-            from .external import (  # pylint:disable=import-error,import-outside-toplevel
-                apfel_utils,
-            )
+            from .external import \
+                apfel_utils  # pylint:disable=import-error,import-outside-toplevel
 
             return apfel_utils.compute_apfel_data(theory, observable, pdf)
 
         elif self.external.upper() == "QCDNUM":
-            from .external import (  # pylint:disable=import-error,import-outside-toplevel
-                qcdnum_utils,
-            )
+            from .external import \
+                qcdnum_utils  # pylint:disable=import-error,import-outside-toplevel
 
             return qcdnum_utils.compute_qcdnum_data(theory, observable, pdf)
 
         elif self.external.lower() == "xspace_bench":
-            from .external import (  # pylint:disable=import-error,import-outside-toplevel
-                xspace_bench_utils,
-            )
+            from .external import \
+                xspace_bench_utils  # pylint:disable=import-error,import-outside-toplevel
 
             return xspace_bench_utils.compute_xspace_bench_data(theory, observable, pdf)
 
+        elif self.external.upper() == "APFEL++":
+            from .external import apfelpy_utils
```

Comment:
please check if and where the pylint fix is needed

Reply:
The `pylint: disable=import-error` is not needed here given that it is instead addressed here:

https://github.com/NNPDF/yadism/blob/4d15c433a55b8622d61db7d869cdc663a0153ab0/src/yadmark/benchmark/external/apfelpy_utils.py#L206